pages/workspace/plugins-page.js

```diff
@@ -0,0 +1,49 @@
+const { BasePage } = require('../base-page');
+const { expect } = require('@playwright/test');
+
+const PLUGIN_URL_PLACEHOLDER = 'Write a plugin URL';
+exports.PluginsPage = class PluginsPage extends BasePage {
+  /**
+   * @param {import('@playwright/test').Page} page
+   */
+  constructor(page) {
+    super(page);
+
+    // POM Locators
+    this.pluginPanel = page.locator('div[class$="workspace_plugins__plugin-management"]');
+    this.closePluginPanelButton = page.locator('button[class$="plugins__close-btn"]');
+    this.pluginUrlInput = page.getByRole('textbox').and(page.getByPlaceholder(PLUGIN_URL_PLACEHOLDER));
+    this.installPluginButton = page.getByRole('button', { name: 'Install' }).first()
+    this.uninstallPluginButton = page.locator('button[class$="plugins__trash-button"]').first();
```

Comment:
In the code talking about "delete" the plugin and it is a "trash" icon too. Maybe it would be better to name it "deletePluginButton" instead of "uninstall".